<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REDTabProgram</class>
 <widget class="QWidget" name="REDTabProgram">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>REDTabProgram</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QListWidget" name="list_programs"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button_delete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_new">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
